core-model/src/main/java/com/sequenceiq/cloudbreak/domain/stack/instance/InstanceGroup.java

```diff
@@ -105,6 +105,12 @@ public void setStack(Stack stack) {
         this.stack = stack;
     }
 
+    public InstanceGroup instanceMetaData(Set<InstanceMetaData> instanceMetaData) {
```

Comment:
this name is more readable and meaningful: `replaceInstanceMetadata` or stuff like this

Reply:
I have followed the same pattern as its used for other entities (like the instanceGroups in Stack object), but that can be done of course